sovtoken/sovtoken/request_handlers/batch_req_handler/token_batch_handler.py

```diff
@@ -0,0 +1,31 @@
+import base58
+from sovtoken.exceptions import TokenValueError
+
+from plenum.server.batch_handlers.batch_request_handler import BatchRequestHandler
+from plenum.server.database_manager import DatabaseManager
+
+
+class TokenBatchHandler(BatchRequestHandler):
+
+    def __init__(self, database_manager: DatabaseManager, ledger_id, utxo_cache):
```

Comment:
I think, that we should register utxo_cache for database_manager into bootstrap stage and don't pass it directly (look at the register_new_store method).